base/compiler/ssair/ir.jl

```diff
@@ -983,14 +997,14 @@ end
 
 function finish_current_bb!(compact, active_bb, old_result_idx=compact.result_idx, unreachable=false)
     if compact.active_result_bb > length(compact.result_bbs)
-        @assert compact.bb_rename[active_bb] == 0
+        #@assert compact.bb_rename[active_bb] == 0
```

Comment:
@Keno should this assert be re-enabled?

Reply:
It'd probably be correct to require this to be zero in both rename arrays, but I'm not entirely sure.